export/src/main/scala/org/clulab/reach/export/serial/SerialJsonOutput.scala

```diff
@@ -0,0 +1,58 @@
+package org.clulab.reach.export.serial
+
+import java.io.File
+import java.nio.file.{Files, Paths}
+import java.util.Date
+import java.util.regex.Pattern
+
+import com.typesafe.scalalogging.LazyLogging
+
+import org.clulab.odin.Mention
+import org.clulab.reach.FriesEntry
+import org.clulab.reach.export.JsonOutputter
+import org.clulab.reach.mentions._
+import org.clulab.reach.mentions.serialization.json._
+
+/**
+  * Defines classes and methods used to output the serial-json output format.
+  *   Written by: Tom Hicks. 2/9/2016.
+  *   Last Modified: Rename output type and this class.
+  */
+class SerialJsonOutput (
+
+  /** The character set encoding to use when writing out the JSON to a file. */
+  encoding: String = "utf-8"
+
+) extends JsonOutputter with LazyLogging {
+
+  /** Returns the given mentions in the serial-json format, as one big string. */
+  override def toJSON (
+    paperId:String,
+    allMentions:Seq[Mention],
+    paperPassages:Seq[FriesEntry],
+    startTime:Date,
+    endTime:Date,
+    outFilePrefix:String
+  ): String = {
+    val mentions = allMentions.map(_.toCorefMention)
```

Comment:
OK, thanks. I just copied this from some existing code...but I forget where now.